library/string.lisp

```diff
@@ -134,6 +136,32 @@ does not have that suffix."
   (define (chars str)
     "Returns an iterator over the characters in `str`."
     (iter:into-iter str))
+  
+  (declare delim-split (Char -> String -> (List String)))
```

Comment:
can we just call it split?

Reply:
I was thinking about that- I'm not sure whether the other split is useful or even desirable.